tests/scripts/run-throughput-perf.py

```diff
@@ -140,6 +173,9 @@ def validate_arg(arg, check):
     validate_arg(arch, lambda item: item in valid_archs[os_group])
     validate_arg(build_type, lambda item: item in valid_build_types)
     validate_arg(run_type, lambda item: item in valid_run_types)
+    validate_arg(iterations, lambda item: isinstance(item, int) or item.isdigit())
```

Comment:
isdigit will return false if the the thing being checked is anything other than a series of 0-9 (ie, only digits. It will return false if iterations passed as a negative number). The only time where item is not a string at this point is the default of 5.